### ISSUE TITLE
Localization Support

### ISSUE THREAD

**Issue by Blueberryy** [CriticalFlaw/TF2HUD.Editor#67]
Very epic tool but really want to translate UI text

**Comment by CriticalFlaw** [comment]
That's a good idea. Parts of the editor will need to rewritten to take strings from a localization file. I could add translations for Russian, but other languages will need to be community contributed.

Also, HUD pages will likely not be translated because the text displayed on them is unique to what's in their schema file and unless the HUD author included their own translations, the page will remain in English.

**Comment by CriticalFlaw** [comment]
Implemented support for multiple languages in: 4d25e26f91e66050197e8c85f65dfcb3915b5ba8. I only included butchered French translations for testing purposes, so that will need to be properly translated by someone familiar with the language.

To add support for other languages, a new resource file will need to be created in `TF2HUD.Editor/src/TF2HUD.Editor/Properties/` called **Resources.xx-XX.resx** where the Xs are replaced with the correct culture code (en-US for English, fr-FR  for French etc.).

I'll create a new ticket to track the translation progress and this one will be closed since the backend code has been added. Thanks!
